src/wallet/wallet.cpp

```diff
@@ -193,7 +405,7 @@ CPubKey CWallet::GetKeyFromKeypath(uint32_t nChange, uint32_t nChild, CKey& secr
         MnemonicContainer mContainer = mnemonicContainer;
         DecryptMnemonicContainer(mContainer);
         SecureVector seed = mContainer.GetSeed();
-        masterKey.SetMaster(&seed[0], seed.size());
+        masterKey.SetMaster(&seed.at(0), seed.size());
```

Comment:
Using std::vector::data() method is the only possible improvement here. Other methods work but at() is just as bad or even worse than []